# Fix crash with gst-inspect
# Chris Lord <chris@openedhand.com>

Upstream-Status: Pending

--- gstreamer-0.10.9/tools/gst-inspect.c.old	2006-09-12 11:56:53.000000000 +0100
+++ gstreamer-0.10.9/tools/gst-inspect.c	2006-09-12 11:57:27.000000000 +0100
@@ -1123,7 +1123,7 @@
   g_option_context_add_main_entries (ctx, options, GETTEXT_PACKAGE);
   g_option_context_add_group (ctx, gst_init_get_option_group ());
   if (!g_option_context_parse (ctx, &argc, &argv, &err)) {
-    g_print ("Error initializing: %s\n", err->message);
+    g_print ("Error initializing: %s\n", err ? err->message : "(null)");
     exit (1);
   }
   g_option_context_free (ctx);
